addons/bus/static/tests/legacy/helpers/mock_server/models/ir_websocket.js

```diff
@@ -11,6 +11,7 @@ patch(MockServer.prototype, {
      * @param imStatusIdsByModel
      */
     _mockIrWebsocket__updatePresence(inactivityPeriod, imStatusIdsByModel) {
+        // gito (to be moved)
```

Comment:
:eyes: is it still needed in the legacy mock server?

Reply:
I believe not, I removed it.

I also see other methods in the legacy mock server like `_mockIrWebsocket__getImStatus` which is overridden in `mail` but never called. Should be possible to remove them too then?

Reply:
Great, the legacy mock server should be removed as soon as possible. I believe the conversion to hoot you completed last week made those methods obsolete, so we can remove them as long as the tests pass.